src/main/resources/jenkins/branch/BranchSource/config.jelly

```diff
@@ -22,19 +22,19 @@
  ~ THE SOFTWARE.
  -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:branch="/lib/branch-api">
-  <branch:rowWrapper name="source">
-    <branch:entryWrapper>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:branch="/lib/branch-api" xmlns:local="local">
+  <div class="tr" name="source">
+    <div>
       <input type="hidden" name="stapler-class" value="${sourceDescriptor.clazz.name}"/>
```

Comment:
OK for now but I have to wonder why this plugin is not using a standard databinding form control from Jenkins core that would work in any version.